extension/src/experiments/columns/collect/index.test.ts

```diff
@@ -28,106 +27,91 @@ describe('collectColumns', () => {
   })
 
   it('should output both params and metrics when both are present', () => {
-    const columns = collectColumns({
-      [EXPERIMENT_WORKSPACE_ID]: {
-        baseline: {
-          data: {
-            metrics: {
-              1: {
-                data: { 2: 3 }
-              }
-            },
-            params: {
-              a: {
-                data: {
-                  b: 'c'
-                }
-              }
+    const columns = collectColumns(
```

Comment:
[F] There should be no behaviour changes here. Just updates to the format of the data.

Reply:
and even then the underlying params/metrics should also be the same